inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/ConceptInstancePanel.html

```diff
@@ -26,6 +26,22 @@
 
         <!-- Instance list -->
         <div wicket:id="instances" class="flex-v-container" style="flex: 0.25 1 auto"></div>
+     	
+     	<!-- Instances in annotated text doc -->
+     	
+     	<div class="scrolling flex-content">
+          <table id="bd" class="table table-striped table-condensed">
+            <wicket:container wicket:id="searchResultGroups">
+              <wicket:container wicket:id="group" />
+            </wicket:container>
+          </table>
+        </div>
+     	
+     	
```

Comment:
Please check your excessive white space before checking code in.

Reply:
WIP : I will see to it.